common/pom.xml

```diff
@@ -31,6 +31,16 @@
             <artifactId>jetty-webapp</artifactId>
             <version>9.4.12.v20180830</version>
         </dependency>
+        <dependency>
+            <groupId>net.jpountz.lz4</groupId>
+            <artifactId>lz4</artifactId>
+            <version>1.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.luben</groupId>
+            <artifactId>zstd-jni</artifactId>
+            <version>1.3.7-3</version>
+        </dependency>
```

Comment:
Out of curiosity, why have we ever looked at `io.netty.handler.codec.compression`?

Reply:
I haven't been working on this, so really don't know if we ever looked at this. In any case we could also add it as third implementation, besides ZSTD and LZ4 and run some testing around it.